tests/integration_tests/test_shell_blade_stiffened_plate_quad.py

```diff
@@ -42,7 +42,7 @@ class ProblemTest(PyTACSTestCase.PyTACSTest):
         "point_load_cgy": 0.500000000000004,
         "point_load_cgz": -0.0035714285714285718,
         "point_load_compliance": 71.87161795633577,
-        "point_load_ks_vmfailure": 0.3304517634314118,
```

Comment:
What's the reason for this large jump in failure values?

Reply:
I guess one of the new failure modes is the active one

Reply:
Can we look a little more into which failure mode is causing the jump? Also have you tried running these changes through a full optimization case? I just want to know that these new checks give reaslistic results

Reply:
Yeah I've run some optimizations on the MACH tutorial wing with the new failure modes, here's an example
![image](https://github.com/smdogroup/tacs/assets/20371123/7b6100c9-23e2-474d-8f9e-1e0b2a4cd50f)

As you can see, not much difference in the final design except a few panels with lower stiffener heights (presumably to avoid stiffener crippling).

Let me do a bit of digging to see which mode is critical in that integration test.

Reply:
Looks like it's the stiffener column buckling that's causing the big rise in this case. If I disable that mode I get failure values slightly higher than the old values and if I also remove the crippling failure mode then I get back the old values pretty much exactly.

Reply:
Ok, that sounds good to me. Since this is a breaking change, we should hold off on merging until we're ready for a new release. Is that alright @A-CGray?

Reply:
I recommend implementing a mode to change the output values to f5 so you can see the failure indices. It would actually be better if you move part of my code for this to the base class I think. Then you could see each failure value as a DV temporarily and figure out which one is critical. See line 840 in `src/constitutive/TACSGPBladeStiffenedShellConstitutive` in my PR